000-pre-vip-config/requires.php

```diff
@@ -21,5 +21,8 @@
 // Load Environment utility class and its helpers, if available
 if ( file_exists( $mu_plugins_base . '/lib/environment/class-environment.php' ) ) {
 	require_once $mu_plugins_base . '/lib/environment/class-environment.php';
```

Comment:
Could drop these then since it's required here: https://github.com/Automattic/vip-go-mu-plugins/blob/master/lib/helpers/environment.php#L6 ?

Reply:
I mean, we can drop it all later, just want to get it deployed first.

Reply:
Oh, I misread your comment a bit, I. think we just want to minimize the chances of breakage but also be explicit as much as possible.